srcpkgs/gns3-gui/template

```diff
@@ -15,7 +15,7 @@ maintainer="Tim Sandquist <tim.sandquist@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://gns3.com"
 distfiles="https://github.com/GNS3/${pkgname}/archive/v${version}.tar.gz"
```

Comment:
```suggestion
homepage="https://gns3.com"
changelog="https://raw.githubusercontent.com/GNS3/gns3-gui/master/CHANGELOG"
distfiles="https://github.com/GNS3/${pkgname}/archive/v${version}.tar.gz"
```

Reply:
I added changelogs from the 2.2 branches of gns3-gui and gns3-server.

Reply:
We should stay with master if possible, because when we update to 2.3/3.0, it will be one less line to change. Is there a reason you chose to go with the 2.2 branch?

Reply:
I thought there were changes in master that weren't in 2.2 but it looks like I was wrong.  I can change it to master.